app/org/sagebionetworks/bridge/validators/SurveyAnswerValidator.java

```diff
@@ -71,55 +73,55 @@ public void validate(Object object, Errors errors) {
-                validateType(errors, (DecimalConstraints) question.getConstraints(), answer.getAnswer());
-                break;
-            case BOOLEAN:
-                validateType(errors, (BooleanConstraints) question.getConstraints(), answer.getAnswer());
-                break;
-            case DATE:
-            case DATETIME:
-                validateType(errors, (TimeBasedConstraints) question.getConstraints(), answer.getAnswer());
-                break;
-            case TIME:
-                validateType(errors, (TimeConstraints) question.getConstraints(), answer.getAnswer());
-                break;
+            String firstAnswer = answer.getAnswers().get(0);
+            if (maybeConstrainedToEnumeratedValue()) {
+                validateType(errors, (MultiValueConstraints) question.getConstraints(), firstAnswer);    
+            } else {
+                Constraints con = question.getConstraints();
+                switch (con.getDataType()) {
+                case DURATION:
+                    validateType(errors, (DurationConstraints) con, firstAnswer);
+                    break;
+                case STRING:
+                    validateType(errors, (StringConstraints) con, firstAnswer);
+                    break;
+                case INTEGER:
+                    validateType(errors, (IntegerConstraints) con, firstAnswer);
+                    break;
+                case DECIMAL:
+                    validateType(errors, (DecimalConstraints) con, firstAnswer);
+                    break;
+                case BOOLEAN:
+                    validateType(errors, (BooleanConstraints) con, firstAnswer);
+                    break;
+                case DATE:
+                case DATETIME:
+                    validateType(errors, (TimeBasedConstraints) con, firstAnswer);
+                    break;
+                case TIME:
+                    validateType(errors, (TimeConstraints) con, firstAnswer);
+                    break;
+                }
             }
         }
         errors.popNestedPath();
     }
-    private boolean isMultiValue() {
+    private boolean maybeConstrainedToEnumeratedValue() {
         return (question.getConstraints() instanceof MultiValueConstraints);
     }
     
-    private boolean expectsMultipleValues() {
-        return isMultiValue() && ((MultiValueConstraints)question.getConstraints()).getAllowMultiple();
+    private boolean allowsMultipleAnswers() {
+        return maybeConstrainedToEnumeratedValue() && ((MultiValueConstraints)question.getConstraints()).getAllowMultiple();
     }
     
     private boolean hasNoAnswer(SurveyAnswer answer) {
-        if (expectsMultipleValues()) {
-            return BridgeUtils.isEmpty(answer.getAnswers());
-        }
-        return StringUtils.isEmpty(answer.getAnswer());
+        return BridgeUtils.isEmpty(answer.getAnswers());
     }
 
     private void validateType(Errors errors, TimeConstraints constraints, String answer) {
```

Comment:
DurationConstraints --> IntegerConstraints, unless it's actually in use, in which case functionally it's the same but with time units only.